nss-day-cohort-39/hello-world-the-globetrotters
# Master + Singapore.html

### Changes
Added the nav bar updates to my singapore.html page.

### Testing
1. Fetch -all 
2. checkout cc-country-page
3. Serve and ensure that the nav bar appears on the Singapore page
